spec/chewy/index/actions_spec.rb

```diff
@@ -802,4 +802,71 @@
       end
     end
   end
+
+  describe 'update_mapping' do
+    before do
+      stub_model(:city)
+      stub_index(:cities) do
+        define_type City
+      end
+      CitiesIndex.create
+    end
+
+    let(:index_name) { CitiesIndex.index_name }
+    let(:body_hash) { {properties: {new_field: {type: :text}}} }
+    let(:unexisting_index) { 'wrong_index' }
+    let(:empty_body_hash) { {} }
+
+    context 'with existing index' do
+      specify do
+        expect(CitiesIndex)
+          .to receive(:update_mapping)
+          .and_call_original
```

Comment:
Since you are calling explicitly `update_mapping`, you can remove all of those `expect(CitiesIndex).to receive(:update_mapping).and_call_original` and you can keep the rest of each spec